README.md

```diff
@@ -193,38 +193,89 @@ const config: Config.InitialOptions = {
 export default config;
 ```
 
-### AskUIAnnotationStepReporter
+### AskUIXRayStepReporter
+
+> ❗️ **IMPORTANT NOTE**: Due to restrictions this reporter only works when you run your workflows in serial (default for AskUI)!
```

Comment:
_in serial_ - is that a common term? Sounds a little weird to me. Maybe _serially_ or _on after another_ instead?